### PR TITLE
[EventHub] drop python 2.7/3.6

### PR DESCRIPTION
**Pull request by swathipil**
fixes: #22267

### CODE REVIEW

File: sdk/eventhub/azure-eventhub-checkpointstoreblob-aio/CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,8 @@
 
 ## 1.1.5 (Unreleased)
```
Comment (rakshith91) [inline sdk/eventhub/azure-eventhub-checkpointstoreblob-aio/CHANGELOG.md:3]: version change to 1.2.0

**Review by rakshith91 — APPROVED** [review]
+1 with version change

File: sdk/eventhub/azure-eventhub-checkpointstoreblob-aio/CHANGELOG.md (hunk 1 above)
Comment (rakshith91) [reply sdk/eventhub/azure-eventhub-checkpointstoreblob-aio/CHANGELOG.md:3]: also, don't forget to change in _version.py  (assuming the build will fail even if you do :) )

Comment (swathipil) [reply sdk/eventhub/azure-eventhub-checkpointstoreblob-aio/CHANGELOG.md:3]: good catch, will do!

**Pull request merged**